tests/misc.rs

```diff
@@ -11,8 +11,23 @@ test! {
   args: ("--list"),
   stdout: "
     Available recipes:
-        foo
-        f   # alias for `foo`
+        foo # [aliases: f]
+  ",
+}
+
+test! {
```

Comment:
I'm actually trying to avoid creating new tests which use the `test!` macro. This should be converted to use the `Test` struct directly. I found that the `test!` macro style was a bit inflexible.